gradle.properties

```diff
@@ -1,7 +1,7 @@
 projectVersion=3.1.1-SNAPSHOT
 projectGroup=io.micronaut.cache
 micronautDocsVersion=2.0.0
-micronautVersion=3.2.3
+micronautVersion=3.3.1
```

Comment:
Bumping the Micronaut version to a new minor version should have required a new minor version of this module too, according to https://github.com/micronaut-projects/micronaut-core/wiki/Module-Upgrade-Strategy

Reply:
thanks, I'll release v3.2.0